easybuild/easyconfigs/z/zeus-mcmc/zeus-mcmc-2.5.4-foss-2022a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonBundle'
+
+name = 'zeus-mcmc'
+version = '2.5.4'
+
+homepage = 'https://zeus-mcmc.readthedocs.io'
+description = "Zeus is a Python implementation of the Ensemble Slice Sampling method."
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('matplotlib', '3.5.2'),
+    ('scikit-learn', '1.1.2'),
+    ('tqdm', '4.64.0'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_LOWER_SOURCE]}
```

Comment:
is this necessary? the default is already `PYPI_SOURCE`, and the extension names are already lower case

Reply:
fixed, it must come from easyconfig I used as a template for this one.